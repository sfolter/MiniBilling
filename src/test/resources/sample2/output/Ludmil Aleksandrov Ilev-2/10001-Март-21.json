{
    "documentDate": "2022-08-10T16:37:26.986158800",
    "documentNumber": "10001",
    "consumer": "Ludmil Aleksandrov Ilev",
    "reference": "2",
    "totalAmount": 243.2,
    "totalAmountWithVat": 291.84,
    "lines": [
        {
            "index": 1,
            "quantity": 106.4,
            "lineStart": "2021-01-01T08:00:00Z",
            "lineEnd": "2021-02-28T23:59:59Z",
            "product": "gas",
            "price": 0.4,
            "priceList": 2,
            "amount": 42.56
        },
        {
            "index": 2,
            "quantity": 45.6,
            "lineStart": "2021-03-01T00:00:00Z",
            "lineEnd": "2021-03-25T13:40:00Z",
            "product": "gas",
            "price": 4.4,
            "priceList": 2,
            "amount": 200.64
        }
    ],
    "vat": [
        {
            "index": 1,
            "lines": [
                1,
                2
            ],
            "percentage": 20,
            "amount": 8.512
        },
        {
            "index": 2,
            "lines": [
                1,
                2
            ],
            "percentage": 20,
            "amount": 40.128
        }
    ]
}