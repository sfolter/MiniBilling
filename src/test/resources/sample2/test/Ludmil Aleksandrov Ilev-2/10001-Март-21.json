{
    "documentDate": "2022-08-11T09:58:56.242814700",
    "documentNumber": "10001",
    "consumer": "Ludmil Aleksandrov Ilev",
    "reference": "2",
    "totalAmount": 127.78,
    "totalAmountWithVat": 153.33,
    "lines": [
        {
            "index": 1,
            "quantity": 106.4,
            "lineStart": "2021-01-01T08:00:00Z",
            "lineEnd": "2021-02-28T23:59:59Z",
            "product": "gas",
            "price": 0.21,
            "priceList": 2,
            "amount": 22.36
        },
        {
            "index": 2,
            "quantity": 45.6,
            "lineStart": "2021-03-01T00:00:00Z",
            "lineEnd": "2021-03-25T13:40:00Z",
            "product": "gas",
            "price": 2.31,
            "priceList": 2,
            "amount": 105.42
        }
    ],
    "vat": [
        {
            "index": 1,
            "lineIndex": 1,
            "percentage": 20,
            "amount": 4.47
        },
        {
            "index": 2,
            "lineIndex": 2,
            "percentage": 20,
            "amount": 21.08
        }
    ]
}