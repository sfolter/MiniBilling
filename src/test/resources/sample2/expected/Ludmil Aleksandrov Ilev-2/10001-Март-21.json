{
  "documentNumber": "10001",
  "consumer": "Ludmil Aleksandrov Ilev",
  "reference": "2",
  "totalAmount": 243.81,
  "lines": [
    {
      "index": 1,
      "quantity": 106.248,
      "lineStart": "2021-01-01T08:00:00Z",
      "lineEnd": "2021-02-28T21:59:59Z",
      "product": "gas",
      "price": 0.4,
      "priceList": 2,
      "amount": 42.5
    },
    {
      "index": 2,
      "quantity": 45.752,
      "lineStart": "2021-02-28T22:00:00Z",
      "lineEnd": "2021-03-25T13:40:00Z",
      "product": "gas",
      "price": 4.4,
      "priceList": 2,
      "amount": 201.31
    }
  ]
}